examples/azure-cni/k8s-scale.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+# some tests set NEW_AGENT_NODE_COUNT in .env files
+ENV_FILE="${CLUSTER_DEFINITION}.env"
+if [ -e "${ENV_FILE}" ]; then
+  source "${ENV_FILE}"
+fi
+
+[[ ! -z "${NEW_AGENT_NODE_COUNT:-}" ]] || (echo "Must specify NEW_AGENT_NODE_COUNT" && exit 1)
+
+OUTPUT="_output/${INSTANCE_NAME}"
+
+# allow nodes to run for a while before scaling
+sleep 180
```

Comment:
after I merge this shouldn't be needed anymore which will help with these tests duration a little.